### ISSUE TITLE
希望支持公式编号和引用功能。

### ISSUE THREAD

**Issue by franklinzhang** [ghosert/cmd-editor#148]
在一个公式内使用 \label{id}给公式编号，然后再在合适的地方\ref{id}即可引用该公式。参见
[Automatic Equation Numbering](http://docs.mathjax.org/en/latest/tex.html#tex-eq-numbers)

> New in MathJax v2.0 is the ability to have equations be numbered automatically. This functionality is **turned off** by default, so that pages don’t change when you update from v1.1 to v2.0, but it is easy to configure MathJax to produce automatic equation numbers by adding:
> 
> ```
> <script type="text/x-mathjax-config">
> MathJax.Hub.Config({
>   TeX: { equationNumbers: { autoNumber: "AMS" } }
> });
> </script>
> ```

**Comment by ghosert** [comment]
已解决，多谢提示。

**Comment by franklinzhang** [comment]
非常感谢 张先生的 辛勤工作

**Comment by ghosert** [comment]
过了一年才修，也是醉了。
